opendj-server-legacy/src/main/java/org/opends/server/backends/pluggable/OnDiskMergeImporter.java

```diff
@@ -1510,7 +1511,7 @@ public MeteredCursor<ByteString, ByteString> flip()
             mmapPosition = region.offset;
             mmap = channel.map(MapMode.READ_ONLY, mmapPosition, Math.min(size.get() - mmapPosition, Integer.MAX_VALUE));
           }
-          final ByteBuffer regionBuffer = mmap.duplicate();
+          final ByteBuffer regionBuffer = ((ByteBuffer)mmap).duplicate();
```

Comment:
Both `MappedByteBuffer` and `ByteBuffer` classes  have the `duplicate` method. `MappedByteBuffer` just extends `ByteBuffer` and uses its `duplicate` methid as superclass method according to [javadoc ](https://docs.oracle.com/javase/8/docs/api/java/nio/MappedByteBuffer.html#methods.inherited.from.class.java.nio.ByteBuffer)